env.default

```diff
@@ -28,6 +28,15 @@ X_FRAME_OPTIONS=DENY
 XSS_PROTECTION=True
 XROBOTSTAG_ENABLED=False
 
+#basket
+
+# See the Basket client docs https://basket-client.readthedocs.io/en/latest/install.html
+BASKET_URL=https://basket-dev.allizom.org/
+BASKET_API_KEY=
+MOFO_NEWSLETTER_SUBSCRIBE_METHOD=BASKET
```

Comment:
Can you move this to a new section that says something like `# Newsletter config:` as it's not really related to Basket specifically?